packages/workflow/src/workflow.ts

```diff
@@ -1347,7 +1374,8 @@ export function setDefaultSignalHandler(handler: DefaultSignalHandler | undefine
  *
  * @param searchAttributes The Record to merge. Use a value of `[]` to clear a Search Attribute.
  */
-export function upsertSearchAttributes(searchAttributes: SearchAttributes): void {
+
```

Comment:
There should be no empty line here. Not sure why the linter isn't complaining.

Reply:
Removed.